src/Management/test/Endpoint.Test/Metrics/Observer/EventCounterListenerTest.cs

```diff
@@ -101,4 +103,41 @@ public void EventCounterListenerGetsMetricsWithExclusionsTest()
             }
         }
     }
+
+    [Fact]
+    public void EventCounterListenerGetsMetricsWithInclusionsTest()
+    {
+        OpenTelemetryMetrics.InstrumentationName = Guid.NewGuid().ToString();
+
+        var inclusions = new List<string>
+        {
+            "cpu-usage"
+        };
+
+        using var listener = new EventCounterListener(new MetricsObserverOptions
+        {
+            IncludedMetrics = inclusions
+        });
+
+        var exporter = new SteeltoeExporter(_scraperOptions);
+        using MeterProvider otelMetrics = GetTestMetrics(null, exporter, null);
+        Task.Delay(2000).Wait();
+
+        var collectionResponse = (SteeltoeCollectionResponse)exporter.CollectionManager.EnterCollectAsync().Result;
```

Comment:
Making the test method `async` and using awaits allows for other tests to run on this thread during the 2s delay.

Reply:
The tests are set to run one at a time. This is there for the asynchrony between this test the Otel library components.

Reply:
It would still block the running of tests in other projects that are not configured to run exclusively. It keeps a thread from the pool busy, which could be used to run unrelated tests on in the meantime.

Reply:
To clarify, I don't mean to remove the wait; just make it an async call. So it suspends this test for 2s without blocking others.

Reply:
Ok fixed.